deeptools/bamCompare.py

```diff
@@ -83,8 +83,13 @@ def getOptionalArgs():
 
     optional.add_argument('--scaleFactorsMethod',
                           help='Method to use to scale the samples. '
-                          'Default "readCount".',
-                          choices=['readCount', 'SES'],
+                          'If scaling is selected, per-sample depth normalization '
+                          'would not be used, instead counts would be scaled between samples '
+                          'to account for sequencing depth differences between the two. '
+                          'As an alternative, scaling can be set to None, and the option '
+                          '--normalizeUsing <method> can be used, to normalize each sample '
+                          'by sequencing depth before comparing them. ',
+                          choices=['readCount', 'SES', 'None'],
```

Comment:
It's a bit unclear what the first sentence here actually means. Do you mean "If a method is selected, then it will be used to compensate for sequencing depth differences between the samples. As an alternative, this can be set to None and the option..."?

Reply:
yes I meant : ``` if --scaleFactorMethod is selected...... alternatively --scaleFactorMethod can be set to None........```

Reply:
Thanks, thought I better double check before I change it :)